docs/pages/bare/migrating-from-expokit.md

```diff
@@ -2,9 +2,9 @@
 title: Migrating from ExpoKit
```

Comment:
we should delete this page - expokit last release was sdk 38, 8 releases / 2 years ago